static/app/constants/index.tsx

```diff
@@ -326,6 +342,15 @@ export const DATA_CATEGORY_INFO = {
     titleName: t('Metrics'),
     uid: 19,
   },
+  // TODO: Deduplicate metrics and metricHour
+  [DataCategoryExact.METRIC_HOUR]: {
+    name: DataCategoryExact.METRIC_HOUR,
+    apiName: 'metrics',
+    plural: 'metrics',
+    displayName: 'metrics',
+    titleName: t('Metrics'),
+    uid: 19,
```

Comment:
something will need to change here

Reply:
@scttcper The data category is being changed from `DataCategory.METRIC_HOUR` to `DataCategory.METRIC_SECOND` to align with how the billing usage will be tracked. However, we will be displaying the usage to customers based on metric hours